src/pages/content-edit/head.js

```diff
@@ -1,439 +1,124 @@
-        };
-
-        if(ctrl.start.ts &&  ctrl.start.ts > ctrl.end.ts) {
-            // Invalid start.
-            ctrl.start = nulledDate(defaultStartTime);
-        }
-
-        ref.update(updated, function() {
-            ctrl.saving = false;
-            m.redraw();
-        });
-    };
-
-    ctrl.clearSchedule = function() {
-        ctrl.start = nulledDate(defaultStartTime);
-        ctrl.end   = nulledDate(defaultEndTime);
-        ctrl.invalidDates = false;
-    };
-
-    ctrl.save = function(opts) {
-        var updated = {};
-
-        ctrl.saving = true;
-        m.redraw();
-
-        updated = {
-            fields : filterHidden( opts.data.fields, opts.hidden ),
-            name   : opts.data.name,
-            slug   : opts.data.slug || null
-        };
-
-        updated = ctrl.addScheduleData(updated);
-
-        return ref.update(updated, function() {
-            ctrl.saving = false;
-            m.redraw();
-        });
-    };
-
-    ctrl.addScheduleData = function(updated) {
-        var startTs = null,
-            endTs = null,
-            startTime = ctrl.start.time || DEFAULT_START_TIME,
-            endTime = ctrl.end.time || DEFAULT_END_TIME;
-
-        if(ctrl.invalidDates) {
-            return updated;
-        }
-
-        if(ctrl.start.date) {
-            startTs = getTimestampFromStr(ctrl.start.date + " " + startTime);
-        }
-
-        if(ctrl.end.date) {
-            endTs = getTimestampFromStr(ctrl.end.date + " " + endTime);
-        }
-
-        updated.published_at = startTs;
-        updated.unpublished_at = endTs;
-
-        if(options.data.published_at && !updated.published_at) {
-            // Publish date was removed. Null it out.
-            updated.published_at = null;
-            updated.published_by = null;
-
-            ctrl.start = nulledDate(defaultStartTime);
-        }
-
-        if(options.data.unpublished_at && !updated.unpublished_at) {
-            // Unpublish date was removed. Null it out.
-            updated.unpublished_at = null;
-            updated.unpublished_by = null;
-
-            ctrl.end = nulledDate(defaultEndTime);
-        }
-
-        return updated;
-    };
-
-
-    ctrl.recalculateTimestamps = function() {
-        var start = ctrl.start,
-            end = ctrl.end;
-
-        if(start.date) {
-            start.ts = getTimestampFromStr(start.date + " " + start.time);
-        }
-        if(end.date) {
-            end.ts = getTimestampFromStr(end.date + " " + end.time);
-        }
-
-        if(start.date && end.date) {
-            ctrl.invalidDates = (start.ts > end.ts);
-        }
-    };  
-
-    ctrl.init();
-}
-
-export function view(ctrl, options) {
-    var status  = "draft",
-        publishTs = options.data.published_at,
-        unpublishTs = options.data.unpublished_at,
-        future  = isFuture(ctrl.start.date + " " + ctrl.start.time),
+export function view(ctrl_unused, options) {
+    var content = options.content,
```

Comment:
To clarify, you want me to align the `=` marks? Sorry, didn't realize that was a required convention.